test/integration/start_stop_delete_test.go

```diff
@@ -352,7 +352,23 @@ func testPulledImages(ctx context.Context, t *testing.T, profile, version string
 	t.Helper()
 	defer PostMortemLogs(t, profile)
 
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "ssh", "-p", profile, "sudo crictl images -o json"))
+	cmd := "sudo crictl images -o json"
```

Comment:
What do you think about using `minikube image list --format=json` instead? We're kind of hacking behind the scenes when this functionality is already built into minikube.

Reply:
@spowelljr thanks for the review and comment!

my thinking was:

i wanted to make a _minimal_ change in the hope that we'll deprecate dockershim at some point soon and then just delete this "workaround" block (as part of a bigger refactor/pruning of related code)
i'd also want to avoid testing a minikube functionality by using another minikube functionality (the "image list" subcommand in this case) as if a test fails, it would not be so obvious what failed specifically and potentially make debugging a bit harder
then i thought about using `docker images` (being the cruntime in this case) directly, but that would require output processing similar to what we do in the `minikube image list`, effectively duplicating that, which made little sense
so, initially, i opted to just keep the `crictl images` that we use in this test...

but, at the end of the day, these are integration tests, so it wouldn't harm much i guess (as long as we keep in mind the above), so i amended the pr using `minikube image list` instead :)

Reply:
local test results:

```
$ make integration -e TEST_ARGS="-minikube-start-args='--driver=kvm2 --container-runtime=docker --alsologtostderr -v=7' -test.run TestStartStop/group/old-k8s-version --cleanup=false"
...
--- PASS: TestStartStop (638.84s)
    --- PASS: TestStartStop/group (0.00s)
        --- PASS: TestStartStop/group/old-k8s-version (638.84s)
            --- PASS: TestStartStop/group/old-k8s-version/serial (638.84s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/FirstStart (160.85s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/DeployApp (9.26s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/EnableAddonWhileActive (0.78s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/Stop (12.21s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/EnableAddonAfterStop (0.21s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/SecondStart (442.66s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/UserAppExistsAfterStop (5.02s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/AddonExistsAfterStop (5.08s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/VerifyKubernetesImages (0.25s)
                --- PASS: TestStartStop/group/old-k8s-version/serial/Pause (2.53s)
PASS
Tests completed in 10m38.840530637s (result code 0)
ok      k8s.io/minikube/test/integration        638.871s
```

Reply:
Not relying on another minikube command is valid, I'm fine with either solution. Once we bump the minimum minikube version I'm fine reverting this PR as well. But if you're happy with it in its current state I'm happy to merge it

Reply:
sure - both would work
i think we can stick with the `minikube image list` for now and switch back to the `crictl images` once we bump the minimum supported k8s version of 1.24+
i've added a TODO comment there, so we don't forget - i think we're good to go with this one ;)